nncf/tensorflow/quantization/algorithm.py

```diff
@@ -92,6 +92,12 @@ def _parse_init_params(self):
         self._batchnorm_adaptation = 'batchnorm_adaptation' in self.config.get('initializer', {})
         self._range_init_params = self._parse_range_init_params()
 
+    def _build_controller(self, model):
```

Comment:
I guess you should replace the `build_controller` method.

Reply:
why? 
I think there should be common logic for setting builder state and algorithm name.
It's already affect ctrl state in TF, because I introduced algorithm name to match arbitrary child state by name, but not by the order in the list

Reply:
I mean that you leave this method without change: https://github.com/openvinotoolkit/nncf/blob/7609fe16f5e406dadb99a21ee6e31179a7dc46ae/nncf/tensorflow/quantization/algorithm.py#L192

Reply:
but I introduced common logic for the ctrl, it will be broken.
```python
    def load_state(self, state: Dict[str, Any]) -> None:
        if self.name in state:
            algo_state = state[self.name]
```